src/pages/BasePersonalisationPage.scss

```diff
@@ -112,4 +112,24 @@
         text-align: center;
     }
 
+    .hasFloatingDoneButton {
+        padding-bottom: 80px;
+
+        .done-button {
+            @include riseIn(1s, 0s, forwards);
+
+            position: fixed;
```

Comment:
Flexboxes would be better for this as they will push down as required. Have a look at how you do a sticky footer with flex boxes (already in our code).